### ISSUE TITLE
Is it possible to disable auto-complete?

### ISSUE THREAD

**Issue by WhiteBlackGoose** [ashmind/SharpLab#876]

**Comment by ashmind** [comment]
Thanks for asking! Not at the moment, but it's not that hard to implement if there is a reason to.
Though I'd rather prioritize fixing any bugs that make it annoying.

**Comment by WhiteBlackGoose** [comment]
Yeah, I agree, but it's probably not an easy task to make it working well. I disable auto-completion in both VS and Rider too, like probably many people. Let's see if this option is interesting for some people aside from me then, I guess

**Comment by tkekalainen** [comment]
I too would like to see an option to disable autocomplete, as it sometimes gets in the way. If possible, could there also be an option to disable the forced autocompletion of words, but still show a list of suggestions, like it does now?

That said, there are a couple of little things which i think would greatly improve the user experience. Here's some problems i've encountered:
1. The autocompletion triggers when starting a string. If you try to type `Console.WriteLine("Hello");` the autocomplete changes it to `Console.WriteLine(arg"Hello");` which causes an error (nevermind that arg doesn't even exist).
2. The same also happens if you close parentheses right after opening them, to call a method with no arguments. `Console.WriteLine()` gets autocompleted to `Console.WriteLine(arg);`, which is not what I wanted. (And again, arg does not exist, so this too causes an error, even if it is correct syntax.)
3. Declaring arrays is also annoying, as typing the `]` in `int []` causes it to change to `int [int]`, again, an error.
4. The autocomplete seems to disregard case. If I write `double` or `string`, it should not change to `Double` or `String`. These two are more cosmetic, but it has "corrected" `c` (a name of a variable) to `C` (a name of a class) for me before. I think it should try for an exact case sensitive match first.

In short:
- Don't autocomplete when starting a string, or when closing parentheses or square brackets right after opening them.
- If the word is an exact case sensitive match, don't try to autocomplete/correct it. 

That's all I can think of right now, but if I find something else, I can mention it later.
